canonical/ubuntu.com
# WD-9393 - move /download/server/thank-you to a separate page

## Done

Move /download/server/thank-you to a separate page

## QA

- [demo link](https://ubuntu-com-13708.demos.haus/download/server)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Click on Download and check that the thank you page is open on a different page

## Issue / Card
[WD-9393](https://warthogs.atlassian.net/browse/WD-9393)


[WD-9393]: https://warthogs.atlassian.net/browse/WD-9393?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
